life/updated/02-life.adoc

```diff
@@ -1568,447 +1566,444 @@ for we were all committed to prison.
 as mine have for myself.
 Be faithful to the spirit of Christ Jesus in your own bosoms,
 and do not overlook little things, for they that are not faithful in a little,
 shall never be made rulers over much.
 Do not exercise yourselves in any matter too high for you,
 but mind the Spirit of Truth in your own hearts,
 and hearken diligently to the voice of the Lord, that your souls may live.
-Keep the Lord always in your remembrance, that you sin not against him;
-remember to keep to the daily cross, which will crucify all the motions of the flesh,
-and keep you alive to God, and near unto him.
-In so doing, you will know his counsel; and seek the kingdom of heaven,
+Keep the Lord always in your remembrance so that you sin not against Him.
+Remember to keep to the daily cross, which will crucify all the motions of the flesh,
+and keep you alive to God, and near unto Him;
+in so doing, you will know His counsel. And seek the kingdom of heaven,
 and the righteousness thereof, above all things in this world,
-and other things shall be added unto you;
+and other things shall be added unto you,
 for I will assure you this is the way that my soul has travailed in,
 and has found favor with God.
 
-One thing more which I have experienced, has been of importance to me:
+One thing more which I have experienced has been of importance to me:
 that in all my afflictions, and pain and sorrow of body or mind,
 I have not had an eye to confide in man, but have applied my heart to the Lord,
-and have poured forth my soul unto him.
-Oh! you Physician of value, that can cure both soul and body;
-you that know better how to administer to my necessity, than I can ask of you;
-from you alone do I look for comfort, for there is none besides you,
+and have poured forth my soul unto Him.
+Oh, you Physician of value, that can cure both soul and body!
+You that know better how to administer to my necessity than I can ask of You!
+From You alone do I look for comfort, for there is none besides You
 that can administer true comfort to me.
-And the Lord in due time, has appeared to my satisfaction, established my goings,
-and kept my feet from falling and my heart from going astray, unto this very day;
-everlasting honor be given unto his name forevermore.
-Amen.
+And the Lord in due time has appeared to my satisfaction, established my goings,
+and kept my feet from falling and my heart from going astray unto this very day.
+Everlasting honor be given unto His name forevermore. Amen.
 
 Since I have seen the good effects of my labor and travail,
-I earnestly beg of the Lord night and day, to do for you, as he has done for me.
+I earnestly beg of the Lord night and day to do for you as He has done for me.
 How have my prayers ascended unto the Lord in public and in private,
-and when my hand has been at labor, and on the highway side.
-Oh! my children, let it not be in vain, for I can truly say,
+and when my hand has been at labor, and on the highway side!
```

Comment:
maybe: and when my hand has been at labor, or I have been travelling upon the road.